### ISSUE TITLE
Map Local/Remote group support

### ISSUE THREAD

**Issue by benjidea** [ProxymanApp/Proxyman#590]
### Proxyman version? (Ex. Proxyman 1.4.3)
Version 2.4.1 (20410)

### macOS Version? (Ex. mac 10.14)
macOS 10.15.6

### Steps to reproduce 
Create multiple Map-Local/Remote entries.

### Expected behavior
It would be nice to group them. So we can enable/disable the mapping for multiple Endpoints with a single click.
(The export/import Feature would also benefit from this)

**Comment by NghiaTranUIT** [comment]
I agree that grouping is useful but I'm not sure if grouping could make sense because Proxyman will evaluate each rule from the top to bottom in order to find the matched rule 🤔 

If I support Grouping feature (Folder), maybe I will introduce the Order Column to explicitly tell the user that Map Local/Remote Tool will try matched rule by the order.

**Comment by benjidea** [comment]
How about tags as an alternative?

**Comment by NghiaTranUIT** [comment]
I would stick to Grouping as a Folder since it's easier to implement than Tag/Label 😄 

I will schedule this feature to Proxyman 2.6.0 (2.5.0 is the next) 👍

**Comment by JasonBoy** [comment]
so is this grouping feature implemented? I still don't see it in 2.9

**Comment by NghiaTranUIT** [comment]
Sorry, but it's delay and has not implemented it yet :/ 

Our team is working on the iOS Proxyman, so this ticket might be rescheduled later. I will push it in the next two releases plan 👍

**Comment by JasonBoy** [comment]
what's even better is to add a mac top menu bar(like charles has one), and we can switch different groups(e.g dev, prod env) quickly

**Comment by NghiaTranUIT** [comment]
I would say grouping by folder is reasonable and it's easier to implement 😄 

For the grouping with environment and Toolbar, we will consider it 👍

**Comment by JasonBoy** [comment]
Just curious, what's the status of this feature? 🤒

**Comment by JasonBoy** [comment]
@NghiaTranUIT since you've already added folder support for `Map Local`, when will this be added for `Map Remote` ?

**Comment by NghiaTranUIT** [comment]
@JasonBoy Sure, I will add the Folder Rule to Map Remote too 👍 At the moment, I only support 3 most used debugging tools (Breakpoint, Map Local, and the Scripting)

Just wondering: how many maps remote rules do you have?

**Comment by JasonBoy** [comment]
almost ~100
